### DISCUSSION TITLE
LGPL-2.1 license is not suitable for games

### DISCUSSION THREAD

**Discussion by RZR001** [JSBSim-Team/jsbsim#976] [General]
Good day,

We wanted to make a sequel to our game using Unreal and JSBSim on consoles, but we realize that LGPL-2.1 does not allow us to do it.
Is there any way to get an MIT license?

**Comment by gallonmate** [comment]
The license is suitable for Unreal made games provided you keep JSBSim as a dynamic library when used with Unreal. See this discussion: [UE5 plugin and JSBSim license/legal clarity + Dynamic Library info](https://github.com/JSBSim-Team/jsbsim/discussions/631).

The UE5 Plugin in the repo meets the license requirements for both Unreal Engine and JSBSim when used for Windows PC. If you can compile your game with JSBSim as a dynamic library and in a way that allows users to modify/change JSBSim in your game by using their own copy of JSBSim, then you'll meet all the requirements of both Unreal Engine and JSBSim. Basically your game will need to allow users to upload a modded JSBSim.

**Comment by seanmcleod** [comment]
@RZR001 is there any particular reason you can't make use of JSBSim as a dynamic library when building for consoles?

Out of interest, what flight dynamics engine do you use in the current version of Frontier Pilot Simulator?

**Comment by RZR001** [comment]
@seanmcleod 
Now in the game we use our own engine, if you would be interested to try it out I will gladly give you a free key

@gallonmate
LGPL-2.1 license that is incompatible with closed source consoles and we can't meet the terms of the 
`and in a way that allows users to modify/change JSBSim in your game by using their own copy of JSBSim`

Is there any way to get an MIT license?
We can even pay, but we don't have any money at all, but for such a cool product we will try to raise personal funds
